tests/testsuite.src/syn_misc.at

```diff
@@ -379,6 +379,7 @@ prog.cob:159: warning: 'PIC-9-SIGNED-DECIMAL' may not be GREATER THAN 99.99
 prog.cob:160: warning: 'PIC-9-SIGNED-DECIMAL' may not be LESS THAN -99.99
 prog.cob:161: warning: 'PIC-9-SIGNED-DECIMAL' may not be LESS THAN -99.99
 prog.cob:162: warning: literal '-99.991' has more decimals than 'PIC-9-SIGNED-DECIMAL'
+prog.cob:164: warning: 'XX' may not be GREATER THAN 99
```

Comment:
I'll check later when you request the review, just wondering - why should this be (only) raised in one case but not in the other?

Reply:
I don't know, I just know that my fix replaces `NOT <` by `>=` and thus triggers the warning in one case.

Reply:
please leave a CHECKME comment in the testsuite source mentioning that we may be able to raise the same for `NOT a < b` as we do for `a NOT < b`